ti-admob/index.js

```diff
@@ -0,0 +1,105 @@
+const mobads = require('com.google.android.gms.ads.MobileAds');
+const Activity = require('android.app.Activity');
+const InterstitialAd = require('com.google.android.gms.ads.InterstitialAd');
+const AdRequest = require('com.google.android.gms.ads.AdRequest');
+const AdListener = require('com.google.android.gms.ads.AdListener');
+
+let hasInitApp = false;
```

Comment:
Remove this, as it's in the global scope and should be in the class scope of `AdView`, e.g. `this.initialized = false` in the constructor.

Reply:
@hansemannn its intentionally in the global scope, as it needs to be done once per session, not once per ad. The class is a single ad